library/repo_test.go

```diff
@@ -290,6 +291,7 @@ func TestLibrary_Repo_String(t *testing.T) {
   Org: %s,
   Private: %t,
   Timeout: %d,
+	Counter: %d,
```

Comment:
Could we fix the spacing here so it's inline with the other fields?

Reply:
Oh weird, my editor rendered it fine. I'll take a look

Reply:
https://github.com/go-vela/types/pull/176/commits/4e63050571a1d705ee5efd1a816b6b17bd2de2f4